server_extensions_extra/src/main/resources/batm-extensions.xml

```diff
@@ -306,6 +306,26 @@
         </ratesource>
         <cryptologo cryptocurrency="POT" file="pot.png"/>
     </extension>
+    <extension class="com.generalbytes.batm.server.extensions.extra.decent.DecentExtension" >
+        <wallet prefix="decentd" name="Decentd">
+            <param name="protocol" />
+            <param name="user" />
+            <param name="password" />
+            <param name="host" />
+            <param name="port" />
+            <param name="accountname" />
+            <cryptocurrency>DCT</cryptocurrency>
+        </wallet>
+        <ratesource prefix="dctfix" name ="Fix Price" >
+            <param name="price" />
+            <cryptocurrency>DCT</cryptocurrency>
+        </ratesource>
+        <ratesource prefix="dctcoinmarketcap" name ="coinmarketcap.com" >
+            <param name="fiatcurrency" />
+            <cryptocurrency>DCT</cryptocurrency>
+        </ratesource>
```

Comment:
Add DCT support to the original CoinmarketcapRateSource present in different extension and this won't be needed.